src/lib/reif.pl

```diff
@@ -1,4 +1,10 @@
-/** Predicates from [*Indexing dif/2*](https://arxiv.org/abs/1607.01590).
+/** Reification is the process of treating the success and failure of a
+predicate as if it were a concrete value. By indexing success and failure to
```

Comment:
Reification means to "turn something into a thing"; not necessarily the truth of a goal, we can also reify backtracking, unification etc.

Reply:
i have made an attempt at correcting it to fit this definition.